philmccarthy/Crate
# Feature/survey buildout

## Is this a fix or a feature?

Feature 

## What is the change/fix?

Built survey functionality to go through survey questions and return a style result for the user. 

## Where should the reviewer start?

code > web > src > modules > user > StyleSurvey.js 

## How should this be tested?

from code/web run `npm start`
from code/api run `npm start`

Go to http://localhost:3000 in browser to signup as a user and take the survey as directed.  The user should be taken through a total of five questions and a results page at the end of the survey that gives the top two styles.

## This PR is related to which issues:
